automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/AsyncImportSpec.scala

```diff
@@ -67,11 +68,16 @@ class AsyncImportSpec
             }
 
             // poll for completion as owner
-            eventually {
-              val resp: HttpResponse = Orchestration.getRequest( s"${workspaceUrl(projectName, workspaceName)}/importPFB/$importJobId")
-              resp.status shouldBe StatusCodes.OK
-              blockForStringBody(resp).parseJson.asJsObject.fields.get("status").value shouldBe JsString("Done")
-            }
+            eventuallyWithEscape{
+            val resp: HttpResponse = Orchestration.getRequest( s"${workspaceUrl(projectName, workspaceName)}/importPFB/$importJobId")
+            resp.status shouldBe StatusCodes.OK
+            val importStatus = blockForStringBody(resp).parseJson.asJsObject.fields.get("status").value
+            assertWithEscape(
+               importStatus shouldBe JsString("Done"),
+              importStatus.equals(JsString("Error")),
+              "Error encountered during import, failing test early"
+            )
+          }
 
             // inspect data entities and confirm correct import as owner
             eventually {
```

Comment:
Should we migrate this eventually call too?

Reply:
For this one and a few others it wasn't obvious to me what a fail condition would be.  I just updated the ones that are checking an import status, since it's easy to know that "Error" means it's not ever going to get to done.  This one is checking all the contents of a map.

Reply:
Makes sense!